Makefile

```diff
@@ -106,6 +106,23 @@ check-format: format
 
 install-core: $(PY27) $(PYPY) $(BEST_PY3) $(TOX)
 
+STACK=$(HOME)/.local/bin/stack
+GHC=$(HOME)/.local/bin/ghc
+SHELLCHECK=$(HOME)/.local/bin/shellcheck
+
+$(STACK):
+	mkdir -p ~/.local/bin
+	curl -L https://www.stackage.org/stack/linux-x86_64 | tar xz --wildcards --strip-components=1 -C $(HOME)/.local/bin '*/stack'
+
+$(GHC): $(STACK)
+	$(STACK) setup
+
+$(SHELLCHECK): $(GHC)
+	$(STACK) install ShellCheck
+
+check-shellcheck: $(SHELLCHECK)
+	shellcheck scripts/*.sh
```

Comment:
For now this is fine, but we might want to revisit this if we get `.sh` files popping up elsewhere for Ophidian.